src/lib/components/SnoozePopup.js

```diff
@@ -83,3 +83,8 @@ export default class SnoozePopup extends React.Component {
     }
   }
 }
+
+SnoozePopup.propTypes = {
```

Comment:
We could, but they aren't used in this class, and they don't throw an error if they aren't listed, so I don't know if we should…